src/datasource.tsx

```diff
@@ -83,7 +82,7 @@ import { PrometheusVariableSupport } from './variables';
 enum PromApplication {
   VictoriaMetrics = 'VictoriaMetrics',
 }
-const ANNOTATION_QUERY_STEP_DEFAULT = '60s';
```

Comment:
I don't think that is good idea export const from main functionality. Maybe it is better to move that const to consts.ts file and export from there?

Reply:
Good idea! Updated